marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/builtins/steps/mapping/entity-services/testCleanMapping.sjs

```diff
@@ -0,0 +1,26 @@
+declareUpdate();
+const test = require("/test/test-helper.xqy");
+
+
+let assertions = [];
+
+assertions.push(test.assertTrue(fn.head(xdmp.eval(
+    "fn.docAvailable('/mappings/OrdersMapping/OrdersMapping-1.mapping.xml') && fn.docAvailable('/mappings/OrdersMapping/OrdersMapping-1.mapping.xml.xslt')",
+    {},{database:xdmp.modulesDatabase()}
+  ))));
+
+
+xdmp.eval('xdmp.documentDelete("/mappings/OrdersMapping/OrdersMapping-1.mapping.json")',
+{},
+{
+  commit: 'auto',
+  update: 'true',
+  isolation: 'different-transaction',
+  ignoreAmps: true
+});
+
+assertions.push(test.assertFalse(fn.head(xdmp.eval(
```

Comment:
I think it's better to make two separate assertions. That's because if one of the documents exists but the other doesn't, the eval will still return false.

Reply:
Ok, will add separate assertions.